docs/source/advanced/boosted_frame.rst

```diff
@@ -45,14 +45,13 @@ the number of required PIC iterations is reduced in the boosted frame:
 
 .. math::
 
-    N_{iterations, boosted} \approx \frac{1}{(1+\beta_{boost})^2 \gamma_{boost}^2} N_{iterations, lab}
+    N_{iterations, boosted} \approx \frac{1}{(1+\beta_b)^2 \gamma_b^2} N_{iterations, lab}
```

Comment:
In the documentation, we use both the notation `\gamma_boost` and `\gamma_b`. This is in order to simplify notations.

Reply:
:white_check_mark: